src/test/regress/citus_tests/join_gen/main.py

```diff
@@ -0,0 +1,65 @@
+import signal
```

Comment:
as I noted, lets change the name of this file to something else. It feels like this is the entry point of the test, which is not